spec/spec_helper.cr

```diff
@@ -25,13 +25,15 @@ macro assert_unauthorized(response)
   fail "expected error response not found" unless found
 end
 
-# Disable all authentication checks by making `DppmRestApi::Actions.has_access?`
-# always return true, then set the authentication back to the default system
-# after yielding to the block.
-def without_authentication!
-  DppmRestApi::Actions.access_filter = ->(_context, _permissions) { true }
-  yield
-  DppmRestApi::Actions.access_filter = DppmRestApi.access_filter
+module DppmRestApi
+  # Disable all authentication checks by making `DppmRestApi::Actions.has_access?`
+  # always return true, then set the authentication back to the default system
+  # after yielding to the block.
+  def self.without_authentication!
```

Comment:
I deleted my comment by mistake because I thought I have submitted it twice :/ (I said it wasn't a good idea to monkey patch be cause the method is only used in specs).

I don't understand, have you an example to show why a normal top_level method can't be used?

Reply:
in the commit I just made -- `protected method 'access_filter=' called for DppmRestApi::Actions:Module`